lib/install.js

```diff
@@ -38,14 +39,40 @@ function install(opts, cb) {
                 _.forEach(methods, function (src, method) {
                     barrier.start(routeName + method);
                     // resolve to the correct file
-                    var sourceFile = path.resolve(opts.basePath, src.source);
+                    var sourceFile = path.resolve(opts.basePath, src.source);                    
                     var route;
-
+                    
                     try {
                         route = {
                             name: routeName,
                             method: method,
-                            func: require(sourceFile)
+                            func: process.env.HOT_RELOAD === 'true'
+                            ? function(req, resp, callback) {
+                                if(typeof process.env.HOT_RELOAD_BASEDIR !== 'undefined') {
+                                    //Delete code loaded from a specific base dir
+                                    Object.keys(require.cache).forEach(function (k) {
+                                        if(k.indexOf(process.env.HOT_RELOAD_BASEDIR) !== -1) {
```

Comment:
`k.includes(process.env.HOT_RELOAD_BASEDIR)` also `k` -> `moduleName` (or what it is)?

Reply:
`k` is the cache key. This is whatever `require` resolved to, which is usually a file path.
Rename to `cacheKey` maybe?

Reply:
👍